docs/usage/configuration.md

```diff
@@ -184,4 +184,77 @@ spec:
 You can set a few additional parameters for the TLS connection: `.tls.authMode`, `tls.permittedPeer`, and `tls.tlsLib`. Refer to the rsyslog documentation for more information on these parameters:
 - [`.tls.authMode`](https://www.rsyslog.com/doc/v8-stable/configuration/modules/omrelp.html#tls-authmode)
 - [`.tls.permittedPeer`](https://www.rsyslog.com/doc/v8-stable/configuration/modules/omrelp.html#tls-permittedpeer)
-- [`.tls.tlsLib`](https://www.rsyslog.com/doc/v8-stable/configuration/modules/imrelp.html#tls-tlslib)
\ No newline at end of file
+- [`.tls.tlsLib`](https://www.rsyslog.com/doc/v8-stable/configuration/modules/imrelp.html#tls-tlslib)
+
+### Configuring the Audit Daemon on the Shoot Nodes
```

Comment:
Does it make sense to also add the new fields in the Shoot spec in https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/b6e6bb8a1091c3443d36efbfda3fb60bd18b23b8/docs/usage/configuration.md#shoot-configuration?